packages/sdks/snippets/angular-ssr/src/app/blog-article/blog-article.component.ts

```diff
@@ -0,0 +1,23 @@
+import { CommonModule } from '@angular/common';
+import { Component } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { Content, type BuilderContent } from '@builder.io/sdk-angular';
+
+@Component({
+  selector: 'app-blog-article',
+  standalone: true,
+  imports: [Content, CommonModule],
+  templateUrl: './blog-article.component.html',
```

Comment:
i just realized you're using a template `HTML` file instead of inlining it here.

Can we inline the `template` here instead? The reason is that we want to provide the entire blueprint in a single code block. By having a separate HTML file, this will require 2 code snippets which will be more work for us to sync when we have the script.

See other components in this codebase to see how we do this

Reply:
@samijaber Sure, Sami, this is very doable. I have done this to make the code clean. But as you suggested, I have reverted it to inline and pushed the code. Kindly review :)